common/src/Microsoft.Azure.IIoT.Serializers.NewtonSoft/src/Default/NewtonSoftJsonSerializer.cs

```diff
@@ -534,6 +534,29 @@ internal sealed class JsonVariantConverter : JsonConverter {
             private readonly NewtonSoftJsonSerializer _serializer;
         }
 
+        /// <summary>
+        /// Json raw converter
+        /// </summary>
+        public sealed class RawJsonConverter : JsonConverter {
```

Comment:
since this is only used in the tests, can we move it in a Tests related class?

Reply:
moved